frontend/apps/docs/next.config.mjs

```diff
@@ -2,9 +2,18 @@ import { createMDX } from 'fumadocs-mdx/next'
 
 const withMDX = createMDX()
 
+function isProductionEnvironment() {
+  if (typeof window !== 'undefined') {
+    return window.location.hostname === 'liam-docs-sigma.vercel.app'
+  }
+  return false
+}
```

Comment:
Can you please explain this to me as I did not understand why this process was necessary?

---

- https://github.com/vercel/next.js/discussions/20650#discussioncomment-4644834
    - In this example, it is always set to `'http://website2.com'`
- https://github.com/vercel/examples/blob/5d40d7600b64f5edb77fd48b4b04fc38e96a894c/solutions/microfrontends/apps/docs/next.config.js#L3
    - The assetPrefix is also a fixed value in the vercel example.

Reply:
I apologize for the lack of explanation🙏. The reason is that setting it to a fixed value causes several path errors when **previewing.** 

ref: https://liam-docs-fk31aq36s-route-06-core.vercel.app/docs

(Additionally, I thought that we wouldn't be able to confirm the changes unless we deployed even during local development.)

So, I decided to only set the assetPrefix for production.

Reply:
@FunamaYukina 
I see. In this case, it would be better to take it out of an environment variable such as `process.env.ASSET_PREFIX`.
By making it variable-dependent instead of environment-dependent, you can usually run the local environment on its own, but debug assetPrefixes in the local environment only when necessary.

ref: https://12factor.net/config

Reply:
I see👍. Configuring ``ASSET_PREFIX`` exclusively for the production environment seems like a more streamlined approach. Thank you for your suggestion🙏. I will implement this change.

Reply:
I've added the ``ASSET_PREFIX`` to Vercel's settings and updated the code accordingly!
https://github.com/liam-hq/liam/pull/369/commits/7cc97e7de8e0960f655363d3473a38d85906bbd1